### PR TITLE
Update DFA.html

### PR DESCRIPTION
**Pull request by rentarded**
Fixes to the sheet in order to fix calculations, as written in https://app.roll20.net/forum/post/4795310/dresden-files-accelerated-errors-in-the-sheet . I'm not a coder, but a friend looked over it and said that it looks like it will function..

### CODE REVIEW

File: DresdenFilesAccelerated/DFA.html [hunk 1]
```diff
@@ -24,45 +25,44 @@
...
-                    <div class="sheet-text-other">Force</div>
+                    <input type="text" placeholder="Force" />
                 </div>
                 
                 <div class="sheet-flexrow">
-                    <button type="roll" class="sheet-button" value="[[4df + @{guile}]]" name="roll_guile"></button>
+                    <button type="roll" class="sheet-button" value="[[4df + @{@guile}]]" name="roll_guile"></button>
                     <input type="number" class="sheet-number" name="attr_guile" title="@{guile}" placeholder="0" />
-                    <div class="sheet-text-other">Guile</div>
-                </div>
```
Comment (Ikidaore) [inline DresdenFilesAccelerated/DFA.html]: Pretty important /div!  A copy-pasta typo, I assume?  =)

**Comment by pjbehr87** [comment]
Like @Ikidaore mentioned above, these changes are removing a required div closing tag. Please fix this before I can accept the PR.

**Comment by rentarded** [comment]
Had no idea. I thought Ikidaore could edit the pull request. My mistake!

**Comment by pjbehr87** [comment]
It turns out, I can actually edit the PRs before pulling them. I think this is a relatively new tool from GitHub, which is why I didn't realize it was an option. Otherwise I would have just made the quick fix myself.

**Pull request merged**